Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabLoader.cpp

```diff
@@ -603,7 +603,8 @@ namespace AzToolsFramework
             }
 
             Instance loadedPrefabInstance;
-            if (!PrefabDomUtils::LoadInstanceFromPrefabDom(loadedPrefabInstance, loadedTemplateDom->get()))
+            PrefabDomUtils::LoadFlags flags = PrefabDomUtils::LoadFlags::ReportDeprecatedComponents;
```

Comment:
nit: This temporary variable declaration can be avoided.

Reply:
I thought it might be more legible to have it as a temporary, and the compiler probably optimizes it anyway. But I don't feel strongly either way. I'll change it.